modules/xmpp/RtxModifier.js

```diff
@@ -0,0 +1,242 @@
+import { getLogger } from "jitsi-meet-logger";
+const logger = getLogger(__filename);
+var transform = require('sdp-transform');
+var SDPUtil = require("./SDPUtil");
+
+/**
+ * Begin helper functions
+ */
+/**
+ * Given a videoMLine, returns a list of the video
+ *  ssrcs (those used to actually send video, not
+ *  any associated secondary streams)
+ * @param videoMLine media line object from transform.parse
+ * @returns list of ssrcs (as strings)
+ */
+function getPrimaryVideoSsrcs (videoMLine) {
+    let videoSsrcs = videoMLine.ssrcs
+        .map(ssrcInfo => ssrcInfo.id)
+        .filter((ssrc, index, array) => array.indexOf(ssrc) === index);
+
+    if (videoMLine.ssrcGroups) {
+        videoMLine.ssrcGroups.forEach((ssrcGroupInfo) => {
+            // Right now, FID groups are the only ones we parse to 
+            //  disqualify streams.  If/when others arise we'll
+            //  need to add support for them here
+            if (ssrcGroupInfo.semantics === "FID") {
+                // secondary FID streams should be filtered out
+                let secondarySsrc = ssrcGroupInfo.ssrcs.split(" ")[1];
```

Comment:
I see that split(" ") then parsing to number is duplicated in many places. Can we wrap that into some util/class ?